### PR TITLE
Add payment controller and service

### PR DESCRIPTION
**Pull request by ReamFOX**

### CODE REVIEW

**Review by Eidenn005 — APPROVED** [review]
Impressed ^_^

File: src/main/java/ua/team3/carsharingservice/exception/InvalidPaymentTypeException.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package ua.team3.carsharingservice.exception;
+
+public class InvalidPaymentTypeException extends RuntimeException {
```
Comment (oksana-miazina) [inline src/main/java/ua/team3/carsharingservice/exception/InvalidPaymentTypeException.java:3]: I'm not sure if you added it to GlobalExceptionHandler

File: src/main/java/ua/team3/carsharingservice/exception/StripeSessionException.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package ua.team3.carsharingservice.exception;
+
+public class StripeSessionException extends RuntimeException {
```
Comment (oksana-miazina) [inline src/main/java/ua/team3/carsharingservice/exception/StripeSessionException.java:3]: And this one

**Review by oksana-miazina — APPROVED** [review]

File: src/main/java/ua/team3/carsharingservice/exception/InvalidPaymentTypeException.java (hunk 1 above)
Comment (ReamFOX) [reply src/main/java/ua/team3/carsharingservice/exception/InvalidPaymentTypeException.java:3]: might have been erased due to conflicts, I'll check later

File: src/main/java/ua/team3/carsharingservice/exception/CustomGlobalExceptionHandler.java [hunk 3]
```diff
@@ -77,6 +77,13 @@ public ResponseEntity<Object> handleForbiddenRequestException(
         return getDefaultTemplate(e, HttpStatus.FORBIDDEN);
     }
 
+    @ExceptionHandler(PaymentProcessedException.class)
+    public ResponseEntity<Object> handlePaymentProcessedException(
+            PaymentProcessedException e) {
+        return getDefaultTemplate(e, HttpStatus.BAD_REQUEST);
+
+    }
```
Comment (boroda4436) [inline src/main/java/ua/team3/carsharingservice/exception/CustomGlobalExceptionHandler.java]:
```suggestion
        return getDefaultTemplate(e, HttpStatus.BAD_REQUEST);
    }
```

**Pull request merged**